spec/system/admin_customize_themes_config_area_spec.rb

```diff
@@ -79,4 +79,21 @@
       I18n.t("admin_js.admin.config_areas.themes_and_components.themes.title"),
     )
   end
+
+  it "allows to control system themes with experimental_system_themes setting" do
+    SiteSetting.experimental_system_themes = ""
+    config_area.visit
+    theme_names = page.all(".theme-card__title").map(&:text)
```

Comment:
Would be good to use page objects more in this spec at some point, also when we have a badge for system themes to update the selector to check that badge is showing too. Can be done later though